integration/bin/tachyon-yarn-submit.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Usage:
+#  tachyon-yarn-submit.sh <numWorkers> <pathHdfs>
+
+function printUsage {
+  echo "Usage: tachyon-yarn-submit.sh <numWorkers> <pathHdfs>"
+  echo -e "  numWorkers        \tNumber of Tachyon workers to launch"
+  echo -e "  pathHdfs          \tpath on HDFS to put tachyon jar and distribute it to YARN"
+  echo
+  echo "Example: ./tachyon-yarn-submit.sh 10 hdfs://localhost:9000/tmp/"
+}
+
+if [ "$#" != 2 ]; then
+  printUsage
+  exit 1
+fi
+
+if [ -z "$HADOOP_HOME" ]; then
+  echo "\$HADOOP_HOME is unset, please set this variable to connect to HDFS and YARN";
+  exit 1
+else
+  echo "Using \$HADOOP_HOME set to '$HADOOP_HOME'";
+fi
+
+SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
+TACHYON_HOME="$(cd "${SCRIPT_DIR}/../.."; pwd)"
+
+source "${SCRIPT_DIR}/common.sh"
+
+NUM_WORKERS=$1
+HDFS_PATH=$2
+TACHYON_TARFILE="tachyon.tar.gz"
+rm -rf $TACHYON_TARFILE
+tar -C $TACHYON_HOME -zcf $TACHYON_TARFILE \
+  assembly/target/tachyon-assemblies-${VERSION}-jar-with-dependencies.jar libexec \
```

Comment:
enumerating the artifacts like this seems error prone (renaming any of these will break the integration) ... in the least, we should check that the expected artifacts exist